### DISCUSSION TITLE
Add new feature allowing map the `CheckList` field of `Card` in trello into taskwarrior

### DISCUSSION THREAD

**Discussion by talkbear** [coddingtonbear/inthe.am#404] [Ideas]
The checklist field of trello card usually can map to a requirement in a project management tool such as ([Aha](https://www.aha.io/)), while card play role of feature, like this:
 
- Cards in Trello(synced to inthe.am) <--> Features in Aha
- Checkitems of Card in Trello <--> Requirements of Feature in Aha

Which makes it neccessary to map the checklist to taskwarrior as for me, i think there are more people have the same pain.

**Now:**
- Cards in Trello <--> Tasks in Inthe.am
- Checklist of Card in Trello <--> No mapped field yet in Inthe.am

**Expected:**
- Cards in Trello <--> Tasks in Inthe.am
- Checklist of Card in Trello <-->Annotations or subtask of Task Inthe.am

@coddingtonbear Please considering this.

**Comment by coddingtonbear** [comment]
I'm afraid I really don't even have enough time to _maintain_ inthe.am, let alone add new features for it that would themselves add to the maintenance burden; so it's extremely unlikely I'm going to pursue this feature request on my own.  Perhaps, though, you could talk somebody else into this or possibly you or somebody else could implement this via an integration that isn't based on Inthe.AM?

**Reply by talkbear** [reply]
Got it. Thanks for the advice. :)
